### PR TITLE
Add invitation to open issues

### PR DESCRIPTION
**Pull request by torgo**
This adds a simple invitation to open issues to the bottom of the  Regexp "Rationale" document.

### CODE REVIEW

File: docs/Correctly-Using-Regular-Expressions-Rationale.md [hunk 1]
```diff
@@ -487,6 +487,10 @@ We would like to thank the following contributors:
 
 For detailed rationale, along with other information such as contributor credits, see [Correctly Using Regular Expressions Rationale](./Correctly-Using-Regular-Expressions-Rationale).
 
+## Contributions and Corrections Welcome
+
+Please feel free to [open an issue](https://github.com/ossf/wg-best-practices-os-developers/issues) with us if you have additions or changes you'd like to see.
```
Comment (torgo) [inline docs/Correctly-Using-Regular-Expressions-Rationale.md:492]:
```suggestion
If you have any additions, changes, or corrections you'd like to suggest, please [open an issue](https://github.com/ossf/wg-best-practices-os-developers/issues). We appreciate your contributions!
```

**Comment by david-a-wheeler** [comment]
See my comment on #524.

**Comment by gkunz** [comment]
Hi @torgo! Are you still interested in getting this in? I support the idea but would ask for incorporating @david-a-wheeler's suggestion.

File: docs/Correctly-Using-Regular-Expressions-Rationale.md [hunk 2]
```diff
@@ -487,6 +487,10 @@ We would like to thank the following contributors:
 
 For detailed rationale, along with other information such as contributor credits, see [Correctly Using Regular Expressions Rationale](./Correctly-Using-Regular-Expressions-Rationale).
 
+## Contributions and Corrections Welcome
+
+If you have any additions, changes, or corrections you'd like to suggest, please [open an issue](https://github.com/ossf/wg-best-practices-os-developers/issues). We appreciate your contributions!
```
Comment (torgo) [inline docs/Correctly-Using-Regular-Expressions-Rationale.md:492]:
```suggestion
If you have any additions, changes, or corrections you'd like to suggest, please [open an issue](https://github.com/ossf/wg-best-practices-os-developers/issues) or [open a pull request](https://github.com/ossf/wg-best-practices-os-developers/pulls). We appreciate your contributions!
```

**Comment by torgo** [comment]
I think this can be merged...

**Pull request merged**